script-library/docker-in-docker-debian.sh

```diff
@@ -7,13 +7,14 @@
 # Docs: https://github.com/microsoft/vscode-dev-containers/blob/main/script-library/docs/docker-in-docker.md
 # Maintainer: The VS Code and Codespaces Teams
 #
-# Syntax: ./docker-in-docker-debian.sh [enable non-root docker access flag] [non-root user] [use moby] [Engine/CLI Version] [Major version for docker-compose]
+# Syntax: ./docker-in-docker-debian.sh [enable non-root docker access flag] [non-root user] [use moby] [Engine/CLI Version] [Major version for docker-compose] [azure DNS override]
```

Comment:
Hi, thanks for the PR. 

However, we have migrated the Features to the [devcontainers/features](https://github.com/devcontainers/) repo. More details in the [announcement issue.](https://github.com/microsoft/vscode-dev-containers/issues/1589)

Do you mind opening the PR in [docker-in-docker](https://github.com/devcontainers/features/tree/main/src/docker-in-docker) Feature instead?

Reply:
Here's the features PR: https://github.com/devcontainers/features/pull/200

As discussed, we would like to make this change here first and follow up with the features change.